procurement_service/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===================
+Procurement service
+===================
+
+* This module allows to generate procurement from confirmed sale orders when
+  the product is a service and is assigned the Buy and Make To Order rules.
+* This procurement when executed, will generate / update a purchase order with
+  the service product.
+* The "Routes" field is always displayed on the product form view, regardless
+  of product type.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/129/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/manufacture/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
```

Comment:
You should add a category 'Usage' and explain how to configure the service product so it generates a PO.
Indeed, I don't think it is obvious that you have to fill the mto and buy route, it deserve an explanation.
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#usage